pegasus/sites.v3/code.org/public/playlab.haml

```diff
@@ -0,0 +1,54 @@
+---
+title: Play Lab
+theme: responsive
+social:
+  "twitter:card": photo
+  "twitter:site": "@codeorg"
+  "twitter:url": "http://code.org/playlab"
+  "twitter:image:src" : "https://code.org/images/mc/mc_social.jpg"
+  "twitter:title": "Minecraft Hour of Code"
+  "twitter:description": "Use blocks of code to take Steve or Alex on an adventure through this Minecraft world"
+  "og:title": "Minecraft Hour of Code"
+  "og:description": "Use blocks of code to take Steve or Alex on an adventure through this Minecraft world"
+  "og:image" : "https://code.org/images/mc/mc_social.jpg"
+  "og:image:width": "1920"
+  "og:image:height": "1080"
+  "og:video": ""
+  "og:video:width": ""
+  "og:video:height": ""
+  "og:video:type": ""
+---
+
+- page_mode = PageMode.get(request)
+
+%link{href: "/css/tutorial-landing.css", rel: "stylesheet"}
+
+%div{style: "clear:both"}
+
+.playlab-container
+  %h1= I18n.t(:playlab_choose_theme)
+  %div.image-collection
+    %a.while-link{href: "https://studio.code.org/s/playlab/stage/1/puzzle/1"}
+      %div.choice
+        %div.choice-container
+          %div.choice-label.purple= I18n.t(:playlab_classic_title)
```

Comment:
If you added new strings, don't forget to

`cd i18n/code.org`
`./sync-pegasus.sh`
`commit all locale files in pegasus/cache/i18n`